src/backup-metadata-generator/Procfile

```diff
@@ -0,0 +1,2 @@
+wait: sleep infinity
+generate-metadata: backup-metadata-generator
```

Comment:
@neil-hickey 

This changes the binary name. Do you know if there are any docs referring to that or folks relying on the binary being named `backup-metadata`?

Reply:
Just discovered that docs refer to some scripts that expect the binary to have this name:
https://github.com/pivotal-cf/docs-tas-kubernetes/blob/789fc839b649e754e0478ff946eafb6f3adb2eb2/bandr-restore-manual.html.md.erb#L229-L230

We can still change it because we have to port over/modify those scripts anyway, but it puts another wrinkle on it.